src/advanced_examples.md

```diff
@@ -484,49 +484,6 @@ on start fn {
 
 This example uses generics to define a Box type and how to work with it at a basic assignment level.
 
-### `loop.ln`
-
-```rust,editable
-from @std/app import start, print, exit
-
-on start {
-  const count = [1, 2, 3, 4, 5]
-  const byTwos = count.map(fn (n: int64): int64 = n * 2)
-  count.map(fn (n: int64) = toString(n)).join(', ').print()
-  byTwos.map(fn (n: int64) = toString(n)).join(', ').print()
-  emit exit 0
-}
-```
-
-Alan does not allow arbitrary loops or recursion, but you can still loop over data. This example shows the primary way to do iteration in Alan using the functional [array api](./builtins/array_api.md) which can be parallelized by the AVM.
```

Comment:
I think this one is still valid to keep. It's how we actually want people to manipulate arrays.